diff --git a/Python_Code/baseline_fine_grained.py b/Python_Code/baseline_fine_grained.py
index e79594c..f0fbd23 100644
--- a/Python_Code/baseline_fine_grained.py
+++ b/Python_Code/baseline_fine_grained.py
@@ -23,7 +23,7 @@ from tensorflow.keras import layers, models
 from tensorflow.keras.models import load_model
 from tensorflow.keras.callbacks import ReduceLROnPlateau, EarlyStopping, ModelCheckpoint
 from dataGenerator import DataGenerator_Coarse, DataGenerator
-from sklearn.metrics import plot_confusion_matrix
+# from sklearn.metrics import plot_confusion_matrix
 from sklearn.metrics import confusion_matrix
 from matplotlib import pyplot as plt
 
